cairosvg/bounding_box.py

```diff
@@ -220,9 +222,7 @@ def bounding_box_text(surface, node):
 
 def angle(bx, by):
     """Get the angle between vector (1,0) and vector (bx,by)."""
-    return fmod(
-        2 * pi + (1 if by > 0 else -1) * acos(bx / sqrt(bx * bx + by * by)),
-        2 * pi)
+    return fmod(tau + copysign(acos(bx / hypot(bx, by)), by), tau)
```

Comment:
I’ve always wanted `copysign`, thanks! So much to learn from the standard library after all these years using Python…